apps/coscrad-frontend/src/components/resources/spatial-features/leaflet/spatial-feature-marker.tsx

```diff
@@ -114,21 +114,18 @@ export const buildSpatialFeatureMarker =
         /**
          * This is a workaround to get the default icon shadows (marker-shadow.png) to
          * display.  There may be a leaflet bug affecting this but I haven't tracked down official
-         * documentation of it but it is reference here: https://stackoverflow.com/a/51232969
+         * documentation of it but it is referenced here: https://stackoverflow.com/a/51232969
          */
         const DefaultIcon = new LeafletIcon({
             iconUrl,
             shadowUrl,
             iconAnchor: [12, 41],
-            shadowAnchor: [20, 95],
+            shadowAnchor: [10, 42],
```

Comment:
This seems to fix the weird position of the drop shadow for the marker on my end.  hopefully that's all that's needed.

Reply:
Yea- this is pretty awkward, but it seems like it is isolated at least.